content/books/java-abc/CLAUDE.md

```
@@ -0,0 +1,182 @@
+- 長い説明は箇条書きや表を活用して読みやすくする
+- 完全初心者向けなので「なぜそうなのか」の理由を丁寧に説明する
+- 難しい概念は身近な例えを使って説明してから技術的な解説に入る
+- 前の章で学んだ内容を参照・活用しながら積み上げ式で解説する
+
+## ディレクトリ構成
+
+書籍は論理的に「部（Part）＞章（Chapter）＞節（Section）」の3階層で構成する。
+ただしディレクトリは「章＞節」の2階層で管理し、部（Part）はディレクトリには表さず `index.md` の目次でまとめる。
+節は1ファイルずつに分割して管理する。
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)